images/packages/binaries/swtpm/werf.inc.yaml

```
@@ -0,0 +1,81 @@
+---
+image: {{ $.ImageType }}/{{ $.ImageName }}
+final: false
+fromImage: builder/scratch
+import:
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /swtpm
+  to: /swtpm
+  before: setup
+
+---
+{{- $version := "0.10.0" }}
+
+{{- $name := print $.ImageName "-dependencies" -}}
+{{- define "$name" -}}
+packages:
+- expect socat twisted-core-tools gcc
+- git pkg-config trousers
+- automake autoconf make gcc
```

```suggestion
- automake autoconf make 
```
remove duplicate gcc

---

Fixed